Mobile App - Netflix Clone

himanshu240601 opened the pull request:
<!--
Thank you for sending the PR! We appreciate you spending the time to work on these changes.

Help us understand your motivation by explaining why you decided to make this change.
You can learn more about contributing to project-guidance here: https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md

Happy Contributing!

-->

### Have you read the [Contributing Guidelines](https://github.com/Kushal997-das/Project-Guidance/blob/main/CONTRIBUTING.md) ?

Yes.

### Description
Hi, I want to contribute to this repo by adding a netflix clone app created using flutter. I put this app under mobile applications/intermediate

### Checklist

- [✓] I've read the contribution guidelines.
- [✓] I've checked the issue list before deciding what to submit.
- [✓] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#779

Kushal997-das commented:
@himanshu240601 Please mention the issue number that you been solved .

himanshu240601 commented:
> @himanshu240601 Please mention the issue number that you been solved .

I've opened an issue, #779 
Kindly check and assign the same.

Kushal997-das reviewed (changes requested):
@himanshu240601 Hello , Your `readme.md` file images are not loading please check it .
Also please try to remove those files those are not really important to this project .

himanshu240601 commented:
> @himanshu240601 Hello , Your `readme.md` file images are not loading please check it . Also please try to remove those files those are not really important to this project .

Hey!, I've made the following changes - 
- Updated the readme.md file in netflix_clone directory
- Added the netflix clone project link in the readme.md in mobile application directory (the link is for the main branch)
- Removed unnecessary files from my project (netflix_clone)

Kushal997-das commented on Mobile Applications/Intermediate/netflix_clone/README.md:
```diff
@@ -0,0 +1,25 @@
+
+### <p align="center">App Screenshots</p>
+
+<p align="center">
+<img src="screenshots/1.jpg" width=200/>
+<img src="screenshots/2.jpg" width=200/>
+<img src="screenshots/3.jpg" width=200/>
+<img src="screenshots/4.jpg" width=200/>
+</p>
+
```
App screenshot are not working in `readme.md` file  [here](https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/README.md)

himanshu240601 replied:
I've checked the readme and it's working fine. All the screenshots are loading correctly.
![Screenshot 2023-01-11 110851](https://user-images.githubusercontent.com/95276862/211726557-48230506-c9fa-4294-9ebd-3a3c7a7bbe84.png)

Kushal997-das replied:
No , it's not [check this](https://github.com/himanshu240601/Project-Guidance/tree/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone)

himanshu240601 replied:
> No , it's not [check this](https://github.com/himanshu240601/Project-Guidance/tree/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone)

I've checked multiple times, there's nothing wrong. I don't know why the screenshots are not working on your side.
Should I remove these from the readme?

Kushal997-das replied:
No do not remove try to fix it . 

This is the [URL](https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/README.md) for same

himanshu240601 replied:
Hey! I've made some changes to the readme file. It should work now. Please check [here!](https://github.com/himanshu240601/Project-Guidance/tree/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone)

Kushal997-das reviewed (changes requested):
Please fix the mentioned things .

Kushal997-das commented on Mobile Applications/Intermediate/netflix_clone/README.md:
```diff
@@ -0,0 +1,8 @@
+## <p align="center">Netflix Clone - Flutter</p>
+
+<p align="center">
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/1.jpg" alt="Image 1" width="200" height="400"/>
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/2.jpg" alt="Image 2" width="200" height="400"/>
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/3.jpg" alt="Image 3" width="200" height="400"/>
+  <img src="https://github.com/himanshu240601/Project-Guidance/blob/mobile-app-project-guidance/Mobile%20Applications/Intermediate/netflix_clone/screenshots/4.jpg" alt="Image 4" width="200" height="400"/>
+</p>
```
![Image not loading](https://user-images.githubusercontent.com/61356005/212120009-b85ba463-26bd-4a5d-8426-74271aba4473.JPG)

himanshu240601 replied:
> ![Image not loading](https://user-images.githubusercontent.com/61356005/212120009-b85ba463-26bd-4a5d-8426-74271aba4473.JPG)

try to check again now!
and if still not working could you please share the URL of the images that you're receiving

Kushal997-das reviewed (changes requested):
@himanshu240601 Images are not loading .

himanshu240601 commented:
great 👍
